_episodes/03-sql-joins-aliases.md

```diff
@@ -8,6 +8,13 @@ objectives:
 - "Employ joins to combine data from two tables."
 - "Apply functions to manipulate individual values."
 - "Employ aliases to assign new names to items in a query."
+keypoints:
+- "Use the `JOIN` command and `ON` or `USING` keywords to combine data from two tables."
```

Comment:
Alternative phrasing?

"Use the `JOIN` command to combine data from two tables."
"Use the `ON` or `USING` keywords to specify which columns link the tables."